<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>791</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="research_btn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_year">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>선택안함</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2016</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2017</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2018</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2019</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2020</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="combo_nation">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>선택안함</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>전국</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>서울특별시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>부산광역시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>대구광역시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>인천광역시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>광주광역시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>대전광역시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>울산광역시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>세종특별자치시</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>경기도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>강원도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>충청북도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>충청남도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>전라북도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>전라남도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>경상북도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>경상남도</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>제주특별자치도</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>지역검색</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>연도검색</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>추가</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>DB선택</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>초등학교 학생수</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>310</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>단위 : 명 / % / 건</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>1인가구비율</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>50</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>혼인건수</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="liner">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>되돌리기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_btn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_graph">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>파이 차트</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_graph_line">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>선그래프</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stick_btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>막대그래프</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
